test/commands/zadd.spec.js

```diff
@@ -34,7 +34,11 @@ describe("The 'zadd' method", function () {
                 client.zrange('infinity', 0, -1, 'WITHSCORES', function (err, res) {
                     assert.equal(res[5], 'inf');
                     assert.equal(res[1], '-inf');
-                    assert.equal(res[3], '9.9999999999999992e+22');
+                    if (process.platform !== 'win32') {
+                        assert.equal(res[3], '9.9999999999999992e+22');
+                    } else {
+                        assert.equal(res[3], '9.9999999999999992e+022');
```

Comment:
@Salakar I guess this is an issue with Redis itself on Windows?

Reply:
Maybe, it was consistent across all Node versions and only occured on Windows.

I did make a note of one strange behaviour  though, it only started  showing  up on the newer Windows version Server 2019, didn't  show on 2018 as far as I remember.

I have a todo already to try repro on my Windows box over the weekend, so will let you know the outcome here.